docker/utility/start_server.sh

```diff
@@ -6,7 +6,7 @@ python /app/manage.py migrate
 if [ "${DJANGO_SETTINGS_MODULE}" = "config.settings.local" ] ; then
     echo "CREATING ADMIN USER FOR TESTING PURPOSES..."
     # Using key error as an indicator for whether or not to run database population and job scheduling scripts
-    echo "from django.contrib.auth import get_user_model; User = get_user_model(); User.objects.create_superuser('admin', 'admin@salesforce.com', 'password')" | python manage.py shell
+    echo "from django.contrib.auth import get_user_model; User = get_user_model(); User.objects.create_superuser('admin', '$SFDX_HUB_USERNAME', 'password')" | python manage.py shell
```

Comment:
I would simply set this to admin@example.com. There's no guarantee that a Salesforce username is a deliverable email address.